utils/sendmail.js

```diff
@@ -24,4 +23,47 @@ const passwordResetEmail = (text, firstName) => `
   </div>
 `;
 
-module.exports = { passwordResetEmail, transport };
+const confirmationMessage = (firstName, token) => `
+  <div className="email" style="
+    border: 1px solid black;
+    padding: 20px;
+    font-family: sans-serif;
+    line-height: 2;
+    font-size: 1.1rem;
+  ">
+    <h2>Hello, ${firstName}</h2>
+    <p>Please confirm email</p>
+    <a href=${process.env.FRONT_BASE_URI}confirmation/${token}>Confirm email</a> 
+  </div>
+`;
+
+const sendEmail = async (message, callback) => {
```

Comment:
Please, refactor it to use promise. Here you have an argument `callback` but you don't pass it on `sendEmail` call. You need to use `new Promise((resolve, reject) => {...})` to properly map callback-based function to promise-based

Reply:
Refactored